src/resources/defi/utils.ts

```diff
@@ -156,22 +168,30 @@ export const parsePositions = (data: AddysPositionsResponse, currency: NativeCur
       },
     }),
     {
+      totalOmittedFromTotals: '0',
       borrows: { amount: '0', display: '0' },
       claimables: { amount: '0', display: '0' },
       deposits: { amount: '0', display: '0' },
     }
   );
 
   const totalAmount = subtract(add(positionsTotals.deposits.amount, positionsTotals.claimables.amount), positionsTotals.borrows.amount);
-
-  const totalDisplay = {
-    amount: totalAmount,
-    display: convertAmountToNativeDisplay(totalAmount, currency),
-  };
+  const omittedTotal = subtract(totalAmount, positionsTotals.totalOmittedFromTotals);
 
   return {
     totals: {
-      total: totalDisplay,
+      total: {
+        amount: totalAmount,
+        display: convertAmountToNativeDisplay(totalAmount, currency),
+      },
+      /**
+       * omittedTotal is the total WITHOUT the positions that are omitted
+       * used for the total wallet balance, the position card still shows the normal total
+       */
+      omittedTotal: {
```

Comment:
found this name (`ommitedTotal`) kinda confusing but couldn't think of anything better, lmk if you think of a more descriptive name

Reply:
i think it would be a lot more clear if we first came up for a name for the types of positions that are being excluded. i'm still unclear on this. what dequalifies a position from having it's value being included in the total wallet balance?